### PR TITLE
Editorial Fixes

### PR DESCRIPTION
**Pull request by efullea**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,12 +183,12 @@
...
       <p>
-        An <dfn>active call</dfn> is a <a>TelephonyCall</a> in the <a href=
-        '#call-state-active'>active</a> state representing a connected call
-        which is and bound to the media input and output devices (e.g.
-        microphone, speaker, tone generator). Note that a call on <a>hold</a>
-        is also active from call signaling point of view, but not bound to
-        media input and output devices.
+        An <dfn>active call</dfn> is a <a>TelephonyCall</a> in the <a
+        class="internalDFN" href='#call-state-active'>active</a> state
+        representing a connected call which is and bound to the media input and
```
Comment (marcoscaceres) [inline index.html]: Grammatical issues "is and bound" (this is also in the original)... should that not be "is bound"?

Comment (marcoscaceres) [inline index.html]: Grammar (sorry, was also wrong in the original!): 

> from call signaling point

should be: 

> from a call signaling point

**Comment by marcoscaceres** [comment]
@efullea let me know if you want to fix the above. Otherwise, just merge an I can take care of those small things.

**Comment by efullea** [comment]
Hi Marcos, I have fixed it. Although in general I think it is better to review just the changes and have separate issues/PR for those errors already in the original text. Thanks for your review.

**Pull request merged**